domains/certificates/Query.API/API.AppTests/UnhealthTests.cs

```diff
@@ -0,0 +1,25 @@
+using System.Net;
+using System.Threading.Tasks;
+using API.AppTests.Infrastructure;
+using Xunit;
+
+namespace API.AppTests;
+
+public class UnhealthTests : IClassFixture<QueryApiWebApplicationFactory>
+{
+    private readonly QueryApiWebApplicationFactory factory;
+
+    public UnhealthTests(QueryApiWebApplicationFactory factory)
+    {
+        this.factory = factory;
```

Comment:
If the `RabbitMqSetup` property is nullable, we could here have a line like `this.factory.RabbitMqSetup = null`, which indicates to the reader that there is no RabbitMQ. What do you think?